src/app-layout/runtime-drawer/index.tsx

```diff
@@ -160,6 +161,14 @@ export const mapRuntimeConfigToAiDrawer = (
           ) : undefined,
         }
       : undefined,
+    exitExpandedModeTrigger: exitExpandedModeTrigger
+      ? {
+          customIcon: exitExpandedModeTrigger.customIcon ? (
+            // eslint-disable-next-line react/no-danger
+            <span style={{ lineHeight: 0 }} dangerouslySetInnerHTML={{ __html: exitExpandedModeTrigger.customIcon }} />
```

Comment:
To reuse number of no-danger exceptions, can we create a reusable utility `customTriggerIcon` and reuse it for both triggers?

Even if it is one line, at least it confines the eslint exception in one place